### PR TITLE
Ftr: Migrate travis Ci to GithubActions

### PR DESCRIPTION
**Pull request by sdttttt**
**What this PR does**:

GithubActions is faster than TravisCI.

**Special notes for your reviewer**:

Please note whether the CI behavior has been changed.

### CODE REVIEW

**Review by AlexStocks — APPROVED** [review]

**Review by watermelo — APPROVED** [review]

**Comment by DogBaoBao** [comment]
The token is expose looks dangerous。

**Comment by sdttttt** [comment]
> The token is expose looks dangerous。

If you want to hide it, you need to set up the secret of the warehouse. ref it through context in the workflow.

File: .github/workflows/github-actions.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        go_version:
+          - 1.13
+        os:
+          - ubuntu-latest
+
+    env:
+      DING_TOKEN: db178ff8c63cdc1b9ae0b86f7c4006ddf35e0249f4ae03441d80e55fdc600419
```
Comment (watermelo) [inline .github/workflows/github-actions.yml:22]: Should not be exposed here, Is there any other way to set it?

Comment (sdttttt) [reply .github/workflows/github-actions.yml:22]: Someone with super access is needed to set up the repository's **secrets**.

**Review by wongoo — APPROVED** [review]

**Pull request merged**